packages/mobile_app/lib/screens/daily/daily_screen.dart

```diff
@@ -189,23 +189,78 @@ class _DailyScreenState extends State<DailyScreen>
                             toShow: toShow,
                           ),
 
-                        const SizedBox(
-                          height: SPACE_BEFORE_AMBIANCE,
-                        ),
+                        /// ambiance
+                        if (isToday &&
+                            dat.user != null &&
+                            dat.user.ambiance != null &&
+                            dat.user.ambiance.isNotEmpty)
+                          ListView.builder(
+                            padding: EdgeInsets.symmetric(vertical: 16.0),
+                            shrinkWrap: true,
+                            scrollDirection: Axis.vertical,
+                            physics: NeverScrollableScrollPhysics(),
+                            itemCount: dat.user.ambiance.length,
+                            itemBuilder: (context, index) {
+                              //
+                              final subject = dat.user.ambiance[index];
 
-                        /// button that says "ambiance (relationship) are not avaible in this version"
-                        const NotAvailableButton(),
+                              //
+                              return AmbiacneSubject(
+                                onOptionsTap: () {
+                                  dat.ambianceChangeSubject = subject;
+                                  focusAmbianceChange();
+                                  //
+                                },
+                                subject: subject,
+                                compatibility: getCompatibility(subject),
+                              );
+                            },
+                          ),
 
-                        const SizedBox(
-                          height: SPACE_AFTER_AMBIANCE,
-                        ),
+                        /// add ambiance
+                        if (isToday)
+                          AddAmbianceButton(
+                            onTap: () => focusAmbianceAdd(),
+                          ),
```

Comment:
Any option for this `AddAmbianceButton` to have bottom margin instead of creating additional `SizeBox`?

Reply:
AddAmbianceButton is a container, can't be constant, and rebuilds every time.